api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -29,8 +32,30 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message Composite {
 }
 
+// Configuration for an extension configuration discovery service with name.
+message DynamicConfig {
+  // The name of the extension configuration. It also serves as a resource name in ExtensionConfigDS.
+  string name = 1 [(validate.rules).string = {min_len: 1}];
+
+  // Configuration source specifier for an extension configuration discovery
+  // service. In case of a failure and without the default configuration, the
+  // listener closes the connections.
```

Comment:
Closes HTTP connection? Should this be responds with 5xx?

Reply:
It is documented similarly for other http config discovery fields